### PR TITLE
Add validating and validated states for source.name field

### PR DESCRIPTION
**Pull request by rvsia**
This add two states for source.name

**Validating**

image

**Validated**

image

**Error message**

image

Example

resolveProps

Any thoughts, @beccaglass ?

### CODE REVIEW

**Comment by beccaglass** [comment]
@rvsia looks great! Can we just change the error text to "That name is taken. Try another."

**Comment by rvsia** [comment]
@beccaglass updated

image

File: packages/sources/package.json [hunk 1]
```diff
@@ -28,8 +28,8 @@
         "awesome-debounce-promise": "^2.1.0"
     },
     "devDependencies": {
-        "@data-driven-forms/pf4-component-mapper": "^2.6.7",
-        "@data-driven-forms/react-form-renderer": "^2.6.7",
+        "@data-driven-forms/pf4-component-mapper": "^2.8.3",
```
Comment (Hyperkid123) [inline packages/sources/package.json:31]: make sure to bump the peer dependency as well (good practices and all that)

Comment (rvsia) [reply packages/sources/package.json:31]: Oh, you are right :sob:

**Pull request merged**